IndexDb.Example/IndexDb.Example.csproj

```diff
@@ -1,15 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">
+﻿<Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">
 
   <PropertyGroup>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net9.0</TargetFramework>
```

Comment:
In the example project I turned to dotnet 9 for the latest features. Or maybe we'd like to use dotnet 9 for the library as well?